diff --git a/source/API/CMakeLists.txt b/source/API/CMakeLists.txt
index e0ecf29..6d2e014 100644
--- a/source/API/CMakeLists.txt
+++ b/source/API/CMakeLists.txt
@@ -185,6 +185,13 @@ if ( CMAKE_SYSTEM_NAME MATCHES "Windows" )
     target_link_libraries(liblldb PRIVATE ${PYTHON_LIBRARIES})
   endif()
 else()
+  target_link_libraries(liblldb INTERFACE ${PYTHON_LIBRARIES})
+  if ( CMAKE_SYSTEM_NAME MATCHES "Darwin" )
+    target_link_options(liblldb PRIVATE -undefined dynamic_lookup)
+  else()
+    target_link_options(liblldb PRIVATE -Wl,--unresolved-symbols=ignore-all)
+  endif()
+
   set_target_properties(liblldb
     PROPERTIES
     OUTPUT_NAME lldb