From 11a0ffb26464e4036a056b05fd2556831c957943 Mon Sep 17 00:00:00 2001
From: Isuru Fernando <isuruf@gmail.com>
Date: Mon, 27 Jul 2026 17:30:33 -0700
Subject: [PATCH 5/7] Run FindPython3 regardless of the options passed

---
 lldb/cmake/modules/FindPythonAndSwig.cmake | 2 --
 1 file changed, 2 deletions(-)

diff --git a/lldb/cmake/modules/FindPythonAndSwig.cmake b/lldb/cmake/modules/FindPythonAndSwig.cmake
index b478038f144d..711e41b36fea 100644
--- a/lldb/cmake/modules/FindPythonAndSwig.cmake
+++ b/lldb/cmake/modules/FindPythonAndSwig.cmake
@@ -37,8 +37,6 @@ macro(FindPython3)
 endmacro()
 
 if(Python3_LIBRARIES AND Python3_INCLUDE_DIRS AND Python3_EXECUTABLE AND LLDB_ENABLE_SWIG)
-  set(PYTHONANDSWIG_FOUND TRUE)
-else()
   if (LLDB_ENABLE_SWIG)
     FindPython3()
   else()
